labs/firepower-restapi-104/firepower-restapi-104.json

```diff
@@ -30,9 +30,11 @@
         "title": "Cisco Security REST API"
     }],
     "files": [{
-        "title": "1.md"
+       "title": "FMC - REST - API - 104",
```

Comment:
Don't need these dashes. Also I believe there could be spelling errors in this file, can you check?